project_scrum/models/project_scrum_meeting.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api, _
+
+
+class ProjectScrumMeeting(models.Model):
+    _name = 'project.scrum.meeting'
+    _description = 'Project Scrum Daily Meetings'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='cascade',
+        index=True,
+        track_visibility='onchange',
+        change_default=True,
+        required=True,
+    )
+    sprint_id = fields.Many2one(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    date_meeting = fields.Date(
+        string='Date',
```

Comment:
There are a lot of places in Odo where I think, why a datetime if only a date is needed. Here I feel the opposite. A meeting starts usually at a specific time, so why not a datetime?